src/main/java/frc/utils/MotorSubsystem.java

```diff
@@ -0,0 +1,11 @@
+package frc.utils;
+
+public abstract class MotorSubsystem extends GBSubsystem {
```

Comment:
remove.

Reply:
if you want to add a subsystem to the brake/coast manager, use the static functions that are there,